sql/core/src/test/scala/org/apache/spark/sql/test/SQLTestUtils.scala

```diff
@@ -227,7 +228,27 @@ private[sql] trait SQLTestUtilsBase
 
   protected override def withSQLConf(pairs: (String, String)*)(f: => Unit): Unit = {
     SparkSession.setActiveSession(spark)
-    super.withSQLConf(pairs: _*)(f)
+    val sc = spark.sparkContext
+    // Set all the specified SQL configs to local properties, so that they can be available at
+    // the executor side.
```

Comment:
SQLConf.get should be available to executors, can you explain why it doesn't work?